protocol-specs/schema/docs/lisk_protocol_specs-definitions-chainstate-properties-chain.md

```diff
@@ -0,0 +1,17 @@
+# Untitled array in LiskProtocolSpec Schema
```

Comment:
do we need to put the schema as `md` file? seems like too many files spread across the folder, and Im not sure who is it useful too